command-line/test/Command/Address/InspectSpec.hs

```diff
@@ -39,6 +39,14 @@ spec = describeCmd [ "address", "inspect" ] $ do
     specInspectAddress ["Shelley", "by pointer"]
         "addr1gw2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzer5ph3wczvf2x4v58t"
 
+    -- reward account: keyhash28
+    specInspectAddress ["Shelley", "by value"]
+        "addr1upshvetj09hxjcm9v9jxgunjv4ehxmr0d3hkcmmvdakx7mq726p32"
+
+    -- reward account: scripthash32
+    specInspectAddress ["Shelley", "by value"]
```

Comment:
Note that you could make the command superfluous and the test stronger by requiring `"script_hash"` to be present in the output (and respectively, "stake_key_hash") for the above.